Settings.js

```diff
@@ -40,6 +40,10 @@ define(function (require /*, exports, module*/) {
 
         function parseSettings(settings) {
             var deferred = Promise.defer();
+            
+            //First things first, lets replace "${project}" with the current project path
+            settings = settings.replace(/\${project}/g,  currentProject.fullPath);
```

Comment:
I think using Mustache here would be really nice because we can add more symbols as we need.  So  would probably something like this:

```javascript
settings = Mustache.render(settings, {project: currentProject.fullPath});
```

Your html annotation changes to {{project}}